src/help.jl

```diff
@@ -3,8 +3,17 @@
 import Markdown
 import REPL
 
+if isdefined( REPL.TerminalMenus, :default_terminal)
+  default_terminal = REPL.TerminalMenus.default_terminal
```

Comment:
Can we also do this? (Might have to use `@static if ...` above)
```suggestion
  import REPL.TerminalMenus: default_terminal
```

Reply:
Yes, this works.